bin/generate-manifest.js

```diff
@@ -0,0 +1,53 @@
+#! /usr/bin/env node
+
+const path = require('path');
+const fs = require('fs');
+const packageMeta = require('../package.json');
+
+const manifest = {
```

Comment:
Hmm.  I worry about not remembering to edit (or not being able to find) this file when we need to update the permissions or change the icons or something.  What do you think about putting a `manifest.in` somewhere and loading that, instead of "hard-coding" it here.